LanguageFeatures/Inline-classes/static_analysis_inline_class_A17_t01.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A compile-time error occurs if an inline class constructor
+/// declares a super parameter
+///
+/// @description Checks that a compile-time error occurs if an inline class
+/// constructor declares a super parameter
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=inline-class
+
+inline class V1 {
+  final int id;
+  V1(this.id);
+}
+
+inline class V2 extends V1 {
```

Comment:
Again, we can't use `extends`, because then every other error will be a derived error. Using `implements V1` should be fine, it is possible, and `super.whatever` should still be an error.

In this case, unfortunately, we can't use any super parameters in an invocation that would actually work if we were calling a constructor of `Object`. 

So we'll probably just have to live with the fact that we can't test that the existence of the super parameter is an error in itself, because it is always also an error because there is no such corresponding parameter of a superconstructor.